src/walk-path.js

```diff
@@ -24,7 +24,6 @@ class WalkPath extends React.Component {
         />
         <Polyline
           className={className}
-          interactive=false
```

Comment:
In fact, my babel compiler seems to think that
```javascript
<T hello=false />
```
is invalid, but
```javascript
<T hello={false} />
```
compiles to
```javascript
React.createElement(T, { hello: false });
```
which looks correct.